packages/core/minos-microservice-networks/minos/networks/decorators/builders.py

```diff
@@ -115,11 +118,26 @@ async def _wrapper(*ag, **kw):
 
             return _wrapper
 
-        return {
+        decorators = {
             decorator: _flatten(decorator, fns)
             for decorator, fns in self._build_all_classes(method_name, **kwargs).items()
         }
 
+        self._validate_not_redefined(decorators)
+
+        return decorators
+
+    # noinspection PyMethodMayBeStatic
+    def _validate_not_redefined(self, decorators: Collection[EnrouteDecorator]) -> None:
+        raw = set(map(lambda obj: (type(obj).__bases__[0], *obj), decorators))
```

Comment:
Could you refactor this into a more verbose code? It gets somewhat difficult to understand :sweat:

Reply:
Yes! I'll replace the lambda function with an inner function, so that I think the variable names will clarify the code. 🙂

Reply:
Done